src/components/Preloader/preloader.module.scss

```diff
@@ -1,90 +1,66 @@
 .preloader {
-  position: fixed;
-  left: 0;
-  top: 0;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  padding: 0;
+  margin: 0;
+  background-color: #1F1939;
+  text-align: center;
+  height: 100vh;
+  font-family: "lato";
+  font-weight: 100;
   width: 100%;
-  height: 100%;
-  z-index: 9999999;
-  background: #4285f4; /*#0000ff;*/ /*#44ce6f;*/
-  /* -webkit-animation: backgroundcolorchange 20s infinite alternate;
-  animation: backgroundcolorchange 20s infinite alternate; */
-
-  .spinner {
-    width: 60px;
-    height: 60px;
-    background-color: #ffffff;
-    margin: 0 auto;
-    -webkit-animation: sk-rotateplane 1.2s infinite alternate;
-    animation: sk-rotateplane 1.2s infinite alternate;
-    position: absolute;
-    right: 0;
-    left: 0;
-    -webkit-box-shadow: 0 0 70px 0 rgba(0, 0, 0, 0.4);
-    box-shadow: 0 0 70px 0 rgba(0, 0, 0, 0.4);
-    top: 50%;
-    margin-top: -75px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding-bottom: 120px;
+  .circle {
+    display: inline-block;
+    width: 15px;
+    height: 15px;
+    background-color: #1F1939;
+    border-radius: 50%;
+    animation: loading 1.5s cubic-bezier(0.8, 0.5, 0.2, 1.4) infinite;
```

Comment:
try reducing it to 1/1.2s cause might be possible that the user might leave the site cause of this.

Reply:
reduced to 1s